lib/bundler/cli/doctor.rb

```diff
@@ -90,5 +92,37 @@ def run
         Bundler.ui.info "No issues found with the installed bundle"
       end
     end
+
+  private
+
+    def check_home_permissions
+      check_for_files_not_owned_by_current_user_but_still_rw
+      check_for_files_not_readable_or_writable
+    end
+
+    def check_for_files_not_owned_by_current_user_but_still_rw
+      return unless any_files_not_owned_by_current_user_but_still_rw?
+      Bundler.ui.warn "Files exist in Bundler home that are owned by another " \
```

Comment:
We want to say which files are not owned by the user or cannot be read/written to. The current error message does not have much value to the user as we're leaving it up to the user to go looking in the bundler dir to see which file(s) have the problem.